### PR TITLE
[CORE-9190] Update to node 16

### PR DESCRIPTION
**Pull request by eliasbraun**
## Node 16 LTS Migration: Update nebenan-helpers
See [JIRA link](https://jira.nebenan.de/browse/CORE-9190) for more infos

### CODE REVIEW

**Review by inomn — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "nebenan-helpers",
-  "version": "5.0.0",
+  "version": "7.0.0",
```
Comment (inomn) [inline package-lock.json]: please run `npm i` to update lock file version

**Review by inomn — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-helpers",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-helpers/issues",
-  "version": "7.0.0",
+  "version": "7.0.1",
```
Comment (peterhass) [inline package.json:10]: no need to republish imo 
the compiled assets shouldn't have changed

**Review by peterhass — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (eliasbraun) [reply package.json:10]: So shall I undo it or do we keep it?

Comment (peterhass) [reply package.json:10]: no point in publishing in my opinion - no difference between compiled assets from 7.0.0 and 7.0.1

**Pull request merged**